tests/integration-tests/processors/processors-test.js

```diff
@@ -71,6 +71,33 @@ describe('Processors api', function() {
                 });
             });
         });
+        describe('POST /v1/processors/{processor_id}/download', () => {
+            it('Update successfully processor content', async function() {
+                let processorId;
+                const uri = '/some-processor-name';
+                const url = 'https://fakesite.com';
+                const processor = {
+                    name: 'some-user-processor',
+                    description: 'This is a description',
+                    type: 'file_download',
+                    file_url: url + uri
+                };
+
+                nock(url).get(uri).reply(200, 'module.exports = 5;');
+                const insertProcessorResponse = await requestSender.createProcessor(processor, validHeaders);
+                should(insertProcessorResponse.statusCode).equal(201);
+                processorId = insertProcessorResponse.body.id;
+
```

Comment:
i would add an assertion here that the processor content is indeed `module.exports = 5;` in order to make it obvious that it successfully updated to `newContent` after re-downloading the file